Desktop/property-pro-app/src/components/navbar/navbar.css

```diff
@@ -0,0 +1,89 @@
+#header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  padding: 20px;
+}
+
+.navbar-link-first-child {
+  display: none;
+}
+.navbar-icon > i {
+  color: white;
+  font-size: 30px;
+}
+.navbar-link-second-child {
+  display: flex;
+  color: white;
+  font-size: 20px;
+  margin-right: 10px;
+}
+
+.companyLogo {
+  display: flex;
+}
+.companyLogo > i {
+  font-size: 30px;
+  color: white;
+}
+.name {
+  color: white;
+  font-style: Georgia (serif);
+  font-size: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .header {
+    display: flex;
+  }
+  .navbar-icon > i {
+    display: none;
+  }
+  .navbar-link-first-child {
+    display: flex;
+    color: black;
+    justify-content: space-between;
+    align-items: center;
+    font-style: arial black;
+    font-size: 20px;
+  }
+
+  .navbar-link {
+    margin-right: 20px;
+    cursor: pointer;
+  }
+  .navbar-link:hover {
+    color: blue;
+  }
+  .navbar-link-1 {
+    margin-right: 20px;
+    cursor: pointer;
+  }
+  .navbar-link-1:hover {
+    color: blue;
+  }
+
+  .companyLogo {
```

Comment:
This is not a good class naming. It should be snake case and seperated with hyphen.